apps/arweave/src/ar.erl

```diff
@@ -586,6 +601,26 @@ start(Config) ->
 	end,
 	start_dependencies().
 
+validate_cm_pool_config(Config) ->
```

Comment:
What happens if a node is a VDF Client *and* a pool client? Will the VDF updates from the VDF server be ignored?

Reply:
Hm, or I guess the VDF Client uses the updates from teh VDF server to validate blocks, but uses the Jobs from the Pool server for mining? What happens if they don't agree? Like if the node receives a Job from the server which it can't validate based on the VDF chain it has received from the VDF Server?


What do you think about only allowing a node to be *either* a Pool Client or a VDF client? And if it's a Pool Client it gets all VDF updates from the Pool Server?

Reply:
Got it. And I see now that you've updated the ar_mining_server to skip all the "validation" steps prior to posting a partial solution. I've added a couple of suggested comments to call this out in the code.